lib/config/config.js

```diff
@@ -107,6 +107,20 @@ module.exports = function (fs, path, url, convict) {
         format: 'nat',
         env: 'IP_RATE_LIMIT_BAN_DURATION_SECONDS'
       },
+      smsRateLimit: {
+        limitIntervalSeconds: {
+          doc: 'Duration of automatic throttling for sms',
+          default: 60 * 15,
+          format: 'nat',
+          env: 'SMS_RATE_LIMIT_INTERVAL_SECONDS'
+        },
+        maxSMSs: {
```

Comment:
I'm not a fan of the capitalisation here, especially as it's all lower case above. I suspect you did it because the pluralised double-s looks funny in lower case, but maybe that's a reason to ditch the plural. `maxSms`?